tests/node/client_test.js

```diff
@@ -14,6 +16,16 @@ ifsc
     assert.equal('KKBK0000261',res['IFSC'])
     assert.equal('110485003',res['MICR'])
     assert.equal('HARYANA',res['STATE'])
+    assert.equal('110485003',res['MICR'])
```

Comment:
add assertion for upi,rtgs, neft and imps.

Reply:
Were covered in the second test, but added anyway.